cst438_gradebook/src/test/java/com/cst438/JunitTestGradebookTests.java

```diff
@@ -1,14 +1,84 @@
-package com.cst438;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import com.cst438.controllers.AssignmentController;
+import com.cst438.domain.Assignment;
+import com.cst438.domain.AssignmentDTO;
+import com.cst438.domain.AssignmentGradeRepository;
+import com.cst438.domain.AssignmentRepository;
+import com.cst438.domain.Course;
+import com.cst438.domain.CourseRepository;
+
+import static org.mockito.Mockito.*;
+
+import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.*;
 
```

Comment:
make sure to have consistency in variable naming you use both asDTO and as, which could be confusing.